src/main/java/com/kodilla/ecommercee/domain/Address.java

```diff
@@ -0,0 +1,74 @@
+package com.kodilla.ecommercee.domain;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+
+@Entity
+public class Address {
+
+    private long id;
+    private String street;
+    private String code;
+    private String city;
+    private User user;
+
+    public Address() {
+    }
+
+    public Address(String street, String code, String city) {
+        this.street = street;
+        this.code = code;
+        this.city = city;
+    }
+
+    @Id
+    @GeneratedValue
+    @Column(name = "ID", unique = true)
```

Comment:
Dodawanie unique dla pola typu id jest niepotrzebne. Sam fakt, ze jest to ID mówi, że musi być unique

Reply:
Zrobione